lib/spack/spack/cmd/install.py

```diff
@@ -81,6 +82,10 @@ def setup_parser(subparser):
     cache_group.add_argument(
         '--no-cache', action='store_false', dest='use_cache', default=True,
         help="do not check for pre-built Spack packages in mirrors")
+    subparser.add_argument(
```

Comment:
@obreitwi I don't immediately see how this makes its way to `Package.install` - typically all command line arguments from the install command are propagated via the `update_kwargs_from_args` method.

Reply:
It is added to the `kwargs` in [Line 42](https://github.com/spack/spack/pull/11107/files/3be6e49cafd507d20ca8edcf72d0b17b72e20fce#diff-55f964577b115bc6cd7b17f0dd2882e2R42), and then propagated by lookup [here](https://github.com/spack/spack/pull/11107/files/3be6e49cafd507d20ca8edcf72d0b17b72e20fce#diff-4c802fc3d026c94cfc483afcaa55d67aR1495).